agent/src/main/java/com/microsoft/applicationinsights/agent/internal/agent/DefaultClassVisitor.java

```diff
@@ -51,7 +52,7 @@ public void visit(int version, int access, String name, String signature, String
     @Override
     public MethodVisitor visitMethod(int access, String name, String desc, String signature, String[] exceptions) {
         MethodVisitor originalMV = super.visitMethod(access, name, desc, signature, exceptions);
-
+        originalMV = new JSRInlinerAdapter(originalMV, access, name, desc, signature, exceptions);
```

Comment:
The next line checks for `null`, which will now always be false. Either that part of the condition is no longer valid and should be removed, or it should be relocated.

Reply:
well yes I think can be moved one step above may be.